src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    action="/search"
+    method="get"
+    class="form"
+    data-qa="big">
```

Comment:
```suggestion
    <form
      action="/search"
      method="get"
      class="form"
      data-qa="big"
    >
```

You should style tags like this, if they have 3 or more props.